### PR TITLE
Add email as label for 2fa

### PR DESCRIPTION
**Pull request by mattsmithies**
As per https://github.com/thecodework/two-factor-authentication/issues/53

### CODE REVIEW

File: src/Http/Controllers/TwoFactorAuthenticationController.php [hunk 1]
```diff
@@ -54,7 +54,10 @@ public function setupTwoFactorAuthentication(Request $request)
             config('2fa-config.digest_algorithm'),
             config('2fa-config.number_of_digits')
         );
-        $totp->setLabel(config('2fa-config.account_name'));
+
+        $totp->setLabel($user->email);
```
Comment (imrealashu) [inline src/Http/Controllers/TwoFactorAuthenticationController.php:58]: What about Laravel systems who don't use emails instead they use phone numbers?

Comment (mattsmithies) [reply src/Http/Controllers/TwoFactorAuthenticationController.php:58]: You're absolutely right, I forked this as a solution to the project I am working on. 

in addition, I've found that even with email added as a label to the TOTP code it wasn't valid with the generated QR code/signature. I believe there needs to be an investigation in the parent project and how to properly integrate this.

This fork of the source is very much in the development/experiment phase at this point.

If I continue with this I'd add additional config parameters to supply a key to match a field in the Users model, defaulting to email. That should provide additional flexibility as needed.

Comment (imrealashu) [reply src/Http/Controllers/TwoFactorAuthenticationController.php:58]: Sure. I will do some rigorous testing on the parent library. 

Regarding the Label, I am still unsure why the 'email' attribute is so discussed. The way I see is that ideally, people can have only one account on a specific platform so providing `APP_NAME` as the label should be fine no?
https://github.com/thecodework/two-factor-authentication/blob/master/src/Http/Controllers/TwoFactorAuthenticationController.php#L58

Please highlight if am I missing anything here.

Comment (mattsmithies) [reply src/Http/Controllers/TwoFactorAuthenticationController.php:58]: Inside of a google authenticator app, there are usually 2 labels, one that describes the service itself and the email of an account.

Granted an average person is likely to have 1 account per service but there is a need for the email or an identifier to be added.

As an example, an individual may have many Google or Slack accounts, or a professional may have to manage multiple accounts to service for their clients.

Comment (imrealashu) [reply src/Http/Controllers/TwoFactorAuthenticationController.php:58]: Understood. Thanks for clarifying. I will check that soon and will consider adding this in the next release. Having said, if you wish to create PR, feel free to do that.  👍

**Pull request merged**